Sources/Run/SwiftPackageInfo.swift

```diff
@@ -147,6 +159,16 @@ extension ParsableCommand {
                 """
       )
     }
+      
+      if let isValidLocalCustomFile, isValidLocalCustomFile == false {
```

Comment:
```suggestion
      if isValidLocalCustomFile == false {
```

Reply:
@marinofelipe his point here is valid, I mean, unwrapping the value before checking it (`if let isValidLocalCustomFile`). As we can see on the line 141, `arguments.xcconfig?` will return an optional, which will make the type of `isValidLocalCustomFile` a `Optional<Bool>`. 
Without that unwrap, he will probably have to check if `isValidLocalCustomFile` is different than `nil` to then check whether it is `false` or not.

Reply:
Thanks for the input @unnamedd :).

`!isValidLocalCustomFile` wouldn't compile without unwrapping the `Optional<Bool>`, but `==` uses an overloaded function that tries to unwrap both values and then compare those, so it should be fine :), and similarly to unwrapping it first, only `Optional.some(false)` will trigger that if statement.